src/sbt-test/jar/classpath-jar/src/main/scala/test/Test.scala

```diff
@@ -3,17 +3,27 @@ package test
 // use dependency library
 
 import akka.actor._
+import akka.pattern._
 import akka.routing.RoundRobinRouter
+import akka.util.Timeout
+import scala.concurrent.duration._
 
 class PrintActor extends Actor{
   def receive = {
-    case msg => println(msg)
+    case msg =>
+      println(msg)
+      context.sender ! "ok"
   }
 }
 
 object Test extends App {
+  implicit val timeout = Timeout(3 seconds)
+
   val system = ActorSystem("testSystem")
+  import system.dispatcher
+
   val router = system.actorOf(Props[PrintActor])
-  router ! "SUCCESS!!"
-  system.shutdown
+  (router ? "SUCCESS!!").foreach { _ =>
+    system.shutdown
+  }
```

Comment:
@muuki88 - this is not related to the RPM symlink issue, but I have intermittent failure on this test today, and hence the fix.

The failure is originally caused by the race condition where the actor system may be shutdown before the actor under test has the chance to print the expected message.